Tests/BowLaws/DivideLaws.swift

```diff
@@ -0,0 +1,22 @@
+import Bow
+import BowGenerators
+import SwiftCheck
+
+public final class DivideLaws<F: Divide & ArbitraryK & EquatableK> {
+    public static func check(
+        isEqual: @escaping (Kind<F, ((Int, Int), Int)>, Kind<F, (Int, (Int, Int))>) -> Bool
+    ) {
+        ContravariantLaws<F>.check()
+        associativity(isEqual: isEqual)
+    }
+    
+    private static func associativity(
+        isEqual: @escaping (Kind<F, ((Int, Int), Int)>, Kind<F, (Int, (Int, Int))>) -> Bool
+    ) {
+        property("Divide - Associativity") <~ forAll { (fa: KindOf<F, Int>) in
+            let a = fa.value.divide(fa.value.divide(fa.value) { ($0, $1) }) { ($0, $1) }
```

Comment:
I usually prefer naming variables instead of using `$0` or `$1` even if they don't have meaningful names.